crates/meroctl/src/cli/context/create.rs

```diff
@@ -0,0 +1,179 @@
+    #[clap(long, short = 'a')]
+    application_id: Option<String>,
+
+    /// Enable dev mode
+    #[clap(long)]
+    dev: bool,
+
+    /// Path to use in dev mode (required in dev mode)
+    #[clap(short, long, requires = "dev")]
+    path: Option<Utf8PathBuf>,
+
+    /// Version of the application (required in dev mode)
+    #[clap(short, long, requires = "dev")]
+    version: Option<Version>,
+}
+impl CreateCommand {
+    pub async fn run(self, root_args: RootArgs) -> eyre::Result<()> {
+        let path = root_args.home.join(&root_args.node_name);
+        if ConfigFile::exists(&path) {
+            if let Ok(config) = ConfigFile::load(&path) {
+                let multiaddr = config
+                    .network
+                    .server
+                    .listen
+                    .first()
+                    .ok_or_else(|| eyre!("No address."))?;
+                let base_url = get_ip(multiaddr)?;
+
+                match self.dev {
+                    true => Ok(link_local_app(base_url, self.path, self.version).await?),
+                    false => Ok(create_context(base_url, self.application_id).await?),
+                }
+            } else {
+                Err(eyre!("Failed to load config file"))
+            }
+        } else {
+            Err(eyre!("Config file does not exist"))
+        }
+    }
+}
+
+async fn create_context(base_url: Url, application_id: Option<String>) -> eyre::Result<()> {
+    let application_id =
+        application_id.ok_or_else(|| eyre!("Application ID is required for starting context"))?;
+
+    app_installed(&base_url, &application_id).await?;
+
+    let url = format!("{}admin-api/contexts-dev", base_url);
+    let client = Client::new();
+    let request = CreateContextRequest { application_id };
+
+    let response = client.post(&url).json(&request).send().await?;
+
+    if response.status().is_success() {
+        let context_response: CreateContextResponse = response.json().await?;
+        let context = context_response.data;
+
+        println!("Context created successfully:");
+        println!("ID: {}", context.id);
+        println!("Application ID: {}", context.application_id);
+    } else {
+        let status = response.status();
+        let error_text = response.text().await?;
+        return Err(eyre!(
+            "Request failed with status: {}. Error: {}",
+            status,
+            error_text
+        ));
+    }
+    Ok(())
+}
+
+async fn app_installed(base_url: &Url, application_id: &String) -> eyre::Result<()> {
+    let url = format!("{}admin-api/applications-dev", base_url);
+    let client = Client::new();
+    let response = client.get(&url).send().await?;
+    if response.status().is_success() {
+        let api_response: ListApplicationsResponse = response.json().await?;
+        let app_list = api_response.data.apps;
+        let x = app_list
+            .iter()
+            .map(|app| app.id.0.clone())
+            .any(|app| app == *application_id);
```

Comment:
this seems wasteful.. the allocation is unnecessary

```suggestion
            .any(|app| &app.id.0 == application_id);
```

or better yet, lean into the available methods on `ApplicationId`, accessing the contained type in a "Newtype" is not recommended

```suggestion
            .any(|app| app.id.as_ref() == application_id);
```